zz/zzcut.sh

```diff
@@ -65,6 +65,7 @@ zzcut ()
 	test -n "$1" || { zztool -e uso cut; return 1; }
 
 	local tipo range ofd codscript qtd_campos only_delim inverte sp rlm
+	local aspas=$(echo "&zwnj;" | zzunescape --html)
```

Comment:
Um artifício sendo usado.
Se o separador for ou conter aspas ( ``` " ``` ) não funcionaria ou funcionaria indevidamente, e mesmo usando o escape haveria problemas e olhe que testei usando até 9 contra-barras.
O que fiz fui substituir a aspas duplas, se houver, no separador por um caractere que não se manifesta visualmente mas é detectável pelo sed, para que posteriormente seja restaurado.
Pode conferir dentro da ``` zzunescape ``` ou ``` zzcodchar ``` o caractere usado.